.github/actions/display-jest-test-summary/action.yml

```
@@ -0,0 +1,32 @@
+name: Display E2E Test Summary
+
+inputs:
+  test_report_path:
+    type: string
+    description: Path of test report JSON file
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Transform Input
+      id: transform_input
+      run: |
+        echo JSON_OUTPUT=$(  jq '{ numTotalTests, numPassedTests, numFailedTests}' ${{ inputs.test_report_path }} ) >> $GITHUB_OUTPUT
+      shell: bash
```

To secure against shell injections, save any `inputs` as `env` first and then reference them that way inside `run:`
```suggestion
    - name: Transform Input
      id: transform_input
      env:
          PATH: ${{ inputs.test_report_path }}
      run: |
        echo JSON_OUTPUT=$(  jq '{ numTotalTests, numPassedTests, numFailedTests}' $PATH ) >> $GITHUB_OUTPUT
      shell: bash
```

---

Great thank you, I have updated the action